src/components/new-safe/CardStepper/index.tsx

```diff
@@ -1,17 +1,22 @@
+import { useState } from 'react'
+import { Box } from '@mui/system'
 import css from './styles.module.css'
 import { Card, LinearProgress, CardHeader, Avatar, Typography, CardContent } from '@mui/material'
 import type { TxStepperProps } from './useCardStepper'
 import { useCardStepper } from './useCardStepper'
 
 export function CardStepper<StepperData>(props: TxStepperProps<StepperData>) {
+  const [progressColor, setProgressColor] = useState('static.primary')
```

Comment:
I'd suggest we add a type to this.

Reply:
I tried using `keyof Palette` which would allow values like `primary` etc. but it seems Mui doesn't provide a type for the dot syntax. I will add a comment as a research task for the future.

Reply:
As these are static values used only in one place. What do you think about importing the colour directly instead of extending the palette?